gulpfile.babel.js

```diff
@@ -41,15 +42,33 @@ gulp.task('mocha', (cb) => {
     });
 });
 
-const DOCKER_RUN = 'docker run -d -p 8080:8080 --name eureka netflixoss/eureka:1.1.147';
+function waitForEureka (cb) {
```

Comment:
I'm a little concerned about this approach because it seems like it could lead to infinite polling if there's just always an error?